docs/howto/create-a-hollow-jar/src/main/resources/project-defaults.yaml

```diff
@@ -0,0 +1,5 @@
+swarm:
```

Comment:
Should this contain the port setting from the removed project-defaults.yml?

Reply:
Not really. The file was (for some reason) not being picked up. The test case was still pointing to 8080.